public_html/templates/base.html

```diff
@@ -38,16 +33,20 @@
 	</section>
 	{% block banners %}
 		<section class="banners">
-			<div class="alert alert-warning alert-dismissable">
-				<button type="button" class="close" data-dismiss="alert" aria-hidden="true">&times;</button>
-				<strong>Hello!</strong> This is a web-version of Plagiabot. Database writes for this tool are <b>now
-					enabled</b>. You can start conducting reviews after you log in.
-				<a href="https://meta.wikimedia.org/wiki/Talk:Community_Tech/Improve_the_plagiarism_detection_bot">
-					You can drop your feedback here</a>. <a href="https://tools.wmflabs.org/copypatrol/">Copypatrol</a>
-				is our live, ready-to-use
-				tool, while <a href="https://tools.wmflabs.org/plagiabot/">Plagiabot</a> is our testing playground.
-				Please do not use Plagiabot to record your edits.
-			</div>
+			{% if staging %}
+				<div class="alert alert-danger text-center">
+					This is a staging environment. For the actual CopyPatrol tool, please use
+					<a href="/copypatrol">tools.wmflabs.org/copypatrol</a>
```

Comment:
Interesting. I don't think there's anything invalid with using a relative path, and in this case it'd be necessary for the link to work on both our local and on Tool Labs – not that we need it to work on our local